src/graph/executor/algo/IsomorExecutor.cpp

```diff
@@ -12,77 +12,61 @@ namespace graph {
 folly::Future<Status> IsomorExecutor::execute() {
    // TODO: Replace the following codes with subgraph matching. Return type.
    SCOPED_TIMER(&execTime_);
-   auto* subgraph = asNode<Subgraph>(node());
+   auto* isomor = asNode<Isomor>(node());
    DataSet ds;
-   ds.colNames = subgraph->colNames();
+   ds.colNames = isomor->colNames();
 
-   uint32_t steps = subgraph->steps();
-   const auto& currentStepVal = ectx_->getValue(subgraph->currentStepVar());
-   DCHECK(currentStepVal.isInt());
-   auto currentStep = currentStepVal.getInt();
-   auto resultVar = subgraph->resultVar();
+   auto iterDV = ectx_->getResult(isomor->getdScanVOut()).iter();
+   auto iterQV = ectx_->getResult(isomor->getqScanVOut()).iter();
+   auto iterDE = ectx_->getResult(isomor->getdScanEOut()).iter();
+   auto iterQE = ectx_->getResult(isomor->getqScanEOut()).iter();
 
-   auto iter = ectx_->getResult(subgraph->inputVar()).iter();
-   auto gnSize = iter->size();
+   unsigned int v_count = iterDV->size();
+   unsigned int l_count = iterDV->size();
+   unsigned int e_count = iterDE->size();
 
-   ResultBuilder builder;
-   builder.value(iter->valuePtr());
+   unsigned int *offset = new unsigned int[v_count + 2];
```

Comment:
(1) Please explain via comments why the size is +2
(2) Please use vector. Don't use new.

Reply:
(1) as the offset is recording the starting position of the node 0 which suppose to be 0, and offset[1] should be the end of position node 0 and start of position node 1. The size of the offset should be at least v_count + 1;
(2) Ok